src/position.cpp

```diff
@@ -478,73 +455,67 @@ void Position::update_slider_blockers(Color c) const {
 
 Bitboard Position::attackers_to(Square s, Bitboard occupied) const {
 
-  return  (pawn_attacks_bb(BLACK, s)       & pieces(WHITE, PAWN))
-        | (pawn_attacks_bb(WHITE, s)       & pieces(BLACK, PAWN))
-        | (attacks_bb<KNIGHT>(s)           & pieces(KNIGHT))
-        | (attacks_bb<  ROOK>(s, occupied) & pieces(  ROOK, QUEEN))
-        | (attacks_bb<BISHOP>(s, occupied) & pieces(BISHOP, QUEEN))
-        | (attacks_bb<KING>(s)             & pieces(KING));
+    return (pawn_attacks_bb(BLACK, s) & pieces(WHITE, PAWN)) | (pawn_attacks_bb(WHITE, s) & pieces(BLACK, PAWN))
```

Comment:
is there a way to relax clang-format for complex multi-line statements? this is pretty annoying not gonna lie

Reply:
we could avoid this by removing the `ColumnLimit`....

Reply:
wouldn't that only impact lines that are longer?